dask/bytes/core.py

```diff
@@ -162,15 +162,18 @@ class OpenFile(object):
         The encoding to use if opened in text mode.
     errors : str or None, optional
         How to handle encoding errors if opened in text mode.
+    newline : None, '', '\n', '\r', or '\r\n'.
```

Comment:
For a fixed set of options, the doc formatting is typically `{None, '', ...}`. And the first one is the default.

Reply:
This is updated now.